src/GridViewItem/style.scss

```diff
@@ -50,8 +44,4 @@
     :not(.pcui-disabled) {
         cursor: pointer;
     }
-}
-
-.pcui-gridview-radiobtn {
```

Comment:
did you mean to remove this?

Reply:
Yeah, that is a redundant bit of styling I introduced in the previous release by mistake :)